content/posts/developing-versionless-graphql-apis.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Developing versionless APIs with GraphQL"
+date: "2020-07-22"
+author: "Bing Xiao"
+github: xo426017
```

Comment:
Since we talked about potentially adding Josh to this as well, I updated the header definition to allow for multiple authors. You'll want to update this definition to look like:

```text
title: "Developing Versionless APIs with GraphQL"
date: "2020-07-22"
authors:
- github: xo426017
   name: "Bing Xiao"
```

You can add as many github/names as needed.

Reply:
Note: When you make this change, if you're still not seeing names/icons next to the blog posts, make sure you get latest from this repository's `develop` branch.

Reply:
Keep a single author format.